pkgs/applications/networking/flexget/default.nix

```diff
@@ -36,22 +36,25 @@ with python'.pkgs;
 
 buildPythonApplication rec {
   pname = "FlexGet";
-  version = "2.17.14";
+  version = "2.17.20";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1wh12nspjzsgb0a7qp67s4k8wssbhhf500s8x8mx2smb1mgy4xzz";
+    sha256 = "1hw16rn3wsc3adkx5ja16cxc9ic7jv2hhkcbxfb7wkym5r4gk7m0";
   };
 
   postPatch = ''
     # remove dependency constraints
     sed 's/==\([0-9]\.\?\)\+//' -i requirements.txt
+    # Set python to 3.6
+    sed -i -e 's/py27/py36/g' setup.cfg
   '';
 
   # ~400 failures
   doCheck = false;
 
   propagatedBuildInputs = [
+    python27
```

Comment:
This does not look right.

Reply:
it needs python2.7 for teh deluge plugin. Taking it out will cause build fail.

Reply:
Then, that plugin has to be removed. We can't have two python versions in here.

Reply:
why should it be removed? Deluge is a different tool.

Reply:
I don't know what deluge is nor do I care. It's simple: Either `python36` or `python27` has to removed from the expression.

Reply:
Okay, can you please once again explain why and where `python27` is needed? Maybe we can find a solution.

Reply:
because it's needed for deluge and the deluge plugin.

Reply:
A little detail would be appreciated. E.g., why did you put it in `propagatedBuildInputs`. Can you point me to some source code that explains this requirement?

Reply:
try to build it without. Then try to build it with it. I don't speak python. I'm just a user of flexget.

Reply:
In that case, you're probably not the right one to do this. An alternative might be to keep using Python 2.7 and override the cherrypy version.